tests/test_legacy_flash_finder.py

```diff
@@ -20,21 +19,3 @@ def test_flash_finder():
         assert os.path.exists(os.path.join(temp_dir, "hiResData.mat"))
         assert os.path.exists(os.path.join(temp_dir, "CameraFrameData.txt"))
         assert os.path.exists(os.path.join(temp_dir, "submissionParameters.txt"))
-
-
-def test_flash_bug1():
```

Comment:
When fixing a bug, the test that revealed the bug should remain in place, but the only line that should get removed is the `with pytest.raises(..):`, and maybe add a comment on the test that this is fixed. That way we can prove to ourselves that the bug was fixed. (The redundant test case can be removed in a later PR).

Reply:
Reverted back to the original test bases for both bugs with the only change being the removal of `with pytest.raises(..):`